docs/source/explore/get-started/install.rst

```diff
@@ -11,30 +11,53 @@ Vircadia has two different installers. The *Client Installer* comes with everyth
 Minimum System Requirements
 --------------------------------
 
-In order to run Vircadia in either VR or Desktop mode, ensure that your computer meets these minimum system requirements: 
+In order to run Vircadia, ensure that your computer meets these minimum system requirements:
 
-* Windows 10, 64-bit or MacOS High Sierra (10.13)
-* 8GB+ RAM i5
-* A wide range of Nvidia, AMD, and Intel graphics cards are supported
+* Operating system:
+
+  * Windows 8.1, 64-bit minimum
+  * Windows 10, 64-bit recommended
+  * MacOS High Sierra (10.13) or newer
+  * Ubuntu 18.04 64 bit or any other modern Linux distribution
```

Comment:
I don't think we should put minimum in front of this, as it is a "requirement" and newer linux distros aren't more recommended.
Similarly I am not sure if commenting on the AppImage makes sense in the system requirements. It is a pretty normal package and we don't do the same on Windows or macOS

Reply:
I don't think that minimum implies that other modern Linux distributions are better than the first, but rather that you must use something along those lines? Even so, I suppose then we should leave off all "Minimum" or "Recommended" for operating systems if we won't do it for all. Otherwise it might look confusing as if we're recommending you use a specific OS over another (e.g. they might think we're saying to use Mac instead of Windows!)